README.md

```diff
@@ -1,2 +1,424 @@
-# jasmine-styleguide
-Suggested best practices when writing Jasmine unit tests.
+# Jasmine Style Guide
+
+## Purpose
+
+The purpose of this style guide is to offer suggested best practices when writing Jasmine unit tests.
+
+## Contributing
+
+This style guide ultimately represents the opinions of its contributors. If you disagree with anything, or wish to add more, please create an issue or submit a pull request. Our goal is to continuously improve the guide and build consensus around it.
+
+## Rules
+
+1. [Speak Human](#speak-human)
+1. [Write _Unit_ Tests](#write-unit-tests)
+1. [Arrange-Act-Assert](#arrange-act-assert)
+1. [Don't Repeat Yourself](#dont-repeat-yourself)
+1. [`this` Is How We Do It](#this-is-how-we-do-it)
+1. [Avoid the `All`s](#avoid-the-alls)
+1. [Be `describe`tive](#be-describetive)
+1. [Write Minimum Passable Tests](#write-minimum-passable-tests)
+
+### Speak Human
+
+Label your test suites (`describe` blocks) and specs (`it` blocks) in a way that clearly conveys the intention of each unit test. Note that the name of each test is the title of its `it` preceded by all its parent `describe` names.
```

Comment:
This is great, thanks @omarmeky! One thing we could add is a section or bullet point on how to word the assertions. Either it('should [do xyz]') or it('[does xyz]'). Not sure if we already have a standard for whether to include the word 'should' or not. Also we want the assertions to be descriptive enough that it's easy to see what is failing, just by looking at the test results.